src/Donations/Micros/PaymentStatus/ThankyouMessage.tsx

```diff
@@ -87,6 +87,28 @@ function ThankyouMessage({
           </>
         )}
 
+        {projectDetails.purpose === "planet-cash" && (
+          <div className="thank-you-purpose">
+            <div className="mt-20">
+              {t("common:pCashTopUpSuccess", {
+                amount: getFormatedCurrency(
+                  i18n.language,
+                  donation.currency,
+                  donation.amount
+                ),
+                frequency: donation.isRecurrent ? t(`${frequency} `) : "",
+              })}
+            </div>
+            <div className="mt-10">{t("common:fundingDonationSuccess")}</div>
+            <div className="go-back">
+              <a href="https://pp.eco/" className="primary-button w-100 mt-30">
```

Comment:
Should be hard wire this URL in our source code? (Compare my other comment above regarding the resource also using this URL.)

Reply:
Removed this button. Now it will be shown only if callback_url is passed in query param.